packages/nestjs/src/integrations/contextlines.ts

```diff
@@ -0,0 +1,21 @@
+import type { IntegrationFn } from '@sentry/core';
+import { defineIntegration } from '@sentry/core';
+import { contextLinesIntegration as originalContextLinesIntegration } from '@sentry/node';
+
+const _contextLinesIntegration = ((options?: { frameContextLines?: number }) => {
+  return originalContextLinesIntegration({
+    ...options,
+    // Nest.js always enabled this, without an easy way for us to detect this
+    // so we just enable it by default
+    // see: https://github.com/nestjs/nest-cli/blob/f5dbb573df1fe103139026a36b6d0efe65e8e985/actions/start.action.ts#L220
+    hasSourceMaps: true,
```

Comment:
m: I'm not sure if this is _really_ always the case. I could imagine a deployment config where a nest app isn't started via the Nest CLI but directly via `node`.In which case, this probably doesn't work as we'd expect it 🤔  Might make sense to check official NestJS deployment guides to be safe.